src/python/module/smartredis/logcontext.py

```diff
@@ -42,13 +41,13 @@ def __init__(self, context):
         self._name = context
 
     @property
-    def _logcontext(self):
+    def _logcontext(self)->t.Type[SRObject]:
```

Comment:
Does should the return type here be `PySRObject`? That appears to be the intended type of `self._srobject` (currently this is getting past the type check bc `PySRObject` aliases to `Any` bc we do not have stubs for `smartredis.smartredisPy`)

Reply:
Good catch!